### PR TITLE
Update design to match collections page

### PR DESCRIPTION
**Pull request by jaronheard**
Updated page design, to match collections page for story collections.

image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,32 +1,131 @@
...
+	  <ul>
+	    <a href="/disaster-resilience/">
+	    	<li class="itemStyle">
+	    		<div class="teamTitleStyle">Hack Oregon 2018</div>
+	    		<h2>Disaster Resilience</h2>
+	    		<a href="https://github.com/hackoregon/disaster-resilience-backend">
+	    			<div class="github">
+	    				<img src="/__assets/github-icon.svg"/>
+	    			</div>
+	    		</a>
```
Comment (DingoEatingFuzz) [inline index.html]: I'm sure this works, but it's technically illegal to put an `a` in an `a`. It might be better to make this a two column flexbox.

**Comment by DingoEatingFuzz** [comment]
🎉 I wildly appreciate the effort to make this look both good and on brand. This is work I've had brewing in the back of my head for awhile now, so I've got some thoughts!

First, on branding, what do you think about differentiating the civic brand from the civic api brand just slightly. Not so much that they look like different experiences, but just enough to make it feel like you're looking under the hood, so to speak.

Second, I love that you are linking back to the Github projects. One of the things that's really been missing is a way to start on the civicplatform site, click through to the apis site, and click through to the github page. That path is huge for transparency and potentially getting involvement outside our own recruitment.

The way the project cards are laid out makes each card look like one big Github link. Reading the source I know that the card links to the API and the octocat links to Github, but that isn't visually discernible. I think even a vertical divider on the left side of the octocat would make it looks like two separate links.

It would also be nice to vertically center the octocat.

The final design critique is the same as the collections page on the platform site: it's very stark. Not sure what to do about that, but I need to say it out loud.

Overall this is such an obvious improvement over the previous design. 

@MikeTheCanuck has been tending this garden, so I'd like to hear his thoughts as well.

---------------------------------------------------------

Bonus content: I have a grander vision of making this the "portal" to all things dev related. A short list being:

1. API documentation
2. Data science methodology documentation
3. Generated "site map" from a meta API that lists all APIs, their year, their city, their repo, and maybe even points of contact.
3. API access workflow (authenticate in some manner, get an API key, etc)
4. Maybe a call to action for getting your data into the civic platform.

File: Dockerfile [hunk 2]
```diff
@@ -5,6 +5,9 @@ VOLUME /usr/share/nginx/html
 VOLUME /etc/nginx
 
 COPY index.html /usr/share/nginx/html/index.html
+COPY index.css /usr/share/nginx/html/__assets/index.css
+COPY github-icon.svg /usr/share/nginx/html/__assets/github-icon.svg
+COPY civic-logo-animated.svg /usr/share/nginx/html/__assets/civic-logo-animated.svg
 COPY strapdown.js /usr/share/nginx/html/__assets/strapdown.js
 COPY strapdown.css /usr/share/nginx/html/__assets/strapdown.css
```
Comment (DingoEatingFuzz) [inline Dockerfile]: Since you removed strapdown from the html page can you also remove it from here?

**Comment by jaronheard** [comment]
Hey, thank you for the feedback @DingoEatingFuzz ! I've pushed a slightly updated design that incorporates your feedback about the card looking like one big GitHub link, and also some feedback from Cat as well. Her feedback boiled down to:
- There are two places we are directing people (APIs and GitHub), and only one of them was obvious
- For non-technical people, the GitHub octo-cat can be disconcerting. Also, it's kind of off-brand.

image

> First, on branding, what do you think about differentiating the civic brand from the civic api brand just slightly. Not so much that they look like different experiences, but just enough to make it feel like you're looking under the hood, so to speak.

Definitely onboard! I'd like to do more in this area, but getting something up is more important at the moment. If you have more thoughts in this area, would love to hear them.

> One of the things that's really been missing is a way to start on the civicplatform site, click through to the apis site, and click through to the github page. That path is huge for transparency and potentially getting involvement outside our own recruitment.

... coming soon to a collections page near you 😉

>  I have a grander vision of making this the "portal" to all things dev related.

Me too, and our visions are super aligned! 📈✨🚀

> Generated "site map" from a meta API that lists all APIs, their year, their city, their repo, and maybe even points of contact.

I've actually done an [API inventory](https://docs.google.com/spreadsheets/d/10UOb8y7MH3o5g1AX2rrM4FPT71uuwiS3u7WngBw6480/edit?usp=sharing) to make moves for this, which is where the 🙈hard-coded🙈 numbers in this latest PR come from.

Another vision I had for using this meta API is to visualize it! I've got some pretty cool network diagram ideas for this.

**Comment by MikeTheCanuck** [comment]
@jaronheard Thank you for providing an entry point from the main-path flow of users, to be able to discover the APIs, docs and code that we worked so hard to make available as part of the platform.  I was hoping that all of my groundwork to pull together the info and refactor some of the DNS would find its way into the grander flow of our resources - mission accomplished!

Love that we have "Get data" and "View code" available for each of the APIs and their endpoints.

Additional feedback:

- What happened to the 2017 APIs?  Right now I'm only seeing "cards" ( am I using the right terms in this instance?) for the 2018 APIs.  [Edit: hrmmm - I accidentally discovered that the right half of the screen scrolls, but there's absolutely no indication that there's more content below the screen - scrolling in the left half - which I did a half-dozen times - provides no cues]
- I agree this still looks stark, cold compared to the civicplatform.org site.  Could we re-use the `.css-vtkknx` top border, and maybe the drifting-molecules background?  If re-using the drifting-molecules feels like not enough distinction from the overall platform, what about using a slight shade of background colour in the main area (e.g. a pale 10-20% green)?


I too have slightly-grander thoughts about how to integrate this content further.  I'd imagined that me getting this endpoint catalog up was just an interim step, and if the content was deemed valuable enough to keep, that it would eventually become another package in the civic repo.  It would probably need a separate `make` script and CD pipeline, but it's really a glorified static site with no specific requirement to standalone as its own repo (that's just its historical underpinnings).

**Pull request merged**